tests/Integration/Behaviour/Features/Scenario/Product/update_suppliers.feature

```diff
@@ -96,10 +96,6 @@ Feature: Update product suppliers from Back Office (BO)
     And product product2 type should be combination
     And product product2 should not have any suppliers assigned
     And product product2 default supplier reference should be empty
-    And product "product2" has following combinations:
```

Comment:
to avoid further questions.
Not really in a scope of PR, but it shouldn't disturb anyone.
Removed this as it is duplicate and redundant (same as in line 92 and no actions have been done, so no point of having it here).